pkg/virt-operator/kubevirt_test.go

```diff
@@ -638,6 +716,19 @@ var _ = Describe("KubeVirt Operator", func() {
 
 	}
 
+	makePodDisruptionBudgetsReady := func() {
```

Comment:
nit: this is a `wait...`, no `make...`

Reply:
Yes, I just didn't want to use a different style, as all the others also follow the pattern `makeXXXReady`...